doc/man/bitcoin-qt.1

```diff
@@ -451,6 +451,9 @@ Set maximum size of high\-priority/low\-fee transactions in bytes
 .IP
 Set lowest fee rate (in BTC/kB) for transactions to be included in block
 creation. (default: 0.00001)
+\fB\-bip148\fR
+.IP
+Enable BIP148 enforcement (see http://www.uasf.co)
```

Comment:
since the primary reason for including the flag would be for people who already know about it and at least think they know what they are doing, an external link might not be needed.